### PR TITLE
Improved live search function

### PR DESCRIPTION
**Pull request by OdedHellman**

### CODE REVIEW

File: superx/information_extractors/item_info_extractor.py [hunk 1]
```diff
@@ -159,7 +159,7 @@ def standardize_weight_name(self, unit_in_hebrew):
             'גרם': ['גרם', 'גרמים'],
             'ליטר': ['ליטר', 'ליטרים', 'ליטר    '],
             'מ"ל': ['מיליליטרים', 'מ"ל', 'מיליליטר'],
-            'אין': ['יחידה', 'לא ידוע', "יח'", "'יח"]
+            'יחידה': ['יחידה', 'לא ידוע', "יח'", "'יח"]
```
Comment (shaibrown5) [inline superx/information_extractors/item_info_extractor.py]: more entries have been added to this list, that do not match יחידה . see my pr. 
Is this neccessary?

File: superx/templates/products_table.html [hunk 2]
```diff
@@ -1,10 +1,8 @@
     {% for product in products %}
     <tr>
        <td> {{ product.name|replace('\'', '') }} </td>
-        {% if product.unit_of_measure == 'לא ידוע' %}
-            <td>1 יחידה </td>
-        {% elif product.unit_of_measure == 'יחידה   '%}
-            <td> 1 יחידה </td>
+        {% if product.unit_of_measure == 'אין' %}
```
Comment (simzacks) [inline superx/templates/products_table.html]: this was changed above to יחידה

File: superx/information_extractors/item_info_extractor.py (hunk 1 above)
Comment (OdedHellman) [reply superx/information_extractors/item_info_extractor.py]: I think that showing the users "אין" as a unit of measure isn't looks well.

Comment (shaibrown5) [reply superx/information_extractors/item_info_extractor.py]: ok got it, mט pr was just added, so go to the script and copy the new varibles into the table

Comment (OdedHellman) [reply superx/information_extractors/item_info_extractor.py]: Done.

**Review by simzacks — APPROVED** [review]

**Pull request merged**